src/components/PlayersList.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import { connect } from 'react-redux';
+// import fetchPlayersReducer from '../reducers/fetchPlayersReducer.js'
+import { fetchPlayers } from '../actions/actions.js';
+import { selectPlayer } from '../actions/actions.js';
+
+class PlayersList extends React.Component {
+    componentDidMount() {
+        this.props.fetchPlayers();
+    }
+
+    componentDidUpdate() {
+        const {players} = this.props;
+        return players.map((player, index)=> {
+            return (
+                <div key={index} onClick={()=> this.props.selectPlayer(player)}>
+                    {player.PLAYER_NAME}
+                </div>
+            ); 
+        });
+    }
+
+    render() {
+        console.log(this.props.players);
+        return (
+            <div>
+                <h3>Players</h3>
+                <div>{this.componentDidUpdate()}</div>
```

Comment:
This is incorrect to use `componentDidUpdate` as `componentDidUpdate` is a reserved function. You can either do one of two things:

a) rename that `componentDidUpdate` function to something else like `renderPlayers`

or 

b) directly put the code inside of the jsx render code

```
   render() {
        const { players } = this.props;

        return (
            <div>
                <h3>Players</h3>
                <div>
                  {players.map((player, index) => 
                    <div key={index} onClick={() => this.props.selectPlayer(player)}>
                      {player.PLAYER_NAME}
                    </div>
                  )}
                </div>
                <br/>
            </div>
        )
    }
```

Up to you, which one you decide.

Reply:
Renamed renderList() to renderPlayers()